### ISSUE TITLE
Redirects from a page we used to own have stopped functioning 

### ISSUE THREAD

**Issue by SimonMurphyDI** [devinit/DIwebsite-redesign#1130]
## I'm submitting a ...
Check one of the following options with "x" and add the appropriate label to the issue as well
<pre><code>
[ X] Bug report 
[ ] Regression (behaviour that used to work and stopped working in a new release)
</code></pre>

**Describe the Issue**
A clear and concise description of what the issue is.

We used to own and manage the website http://www.globalhumitarianassistance.org/. We decided to close this site and migrate some content to the DI site. I think we also created a number of specific redirects that arent now functioning. 

**To Reproduce**
Replace the content below with the steps to reproduce the behaviour.
1. Go to '...' http://www.globalhumitarianassistance.org/
2. No redirect!

**Expected Behaviour**
A clear and concise description of what you expected to happen

Please can all pages from that domain link to: https://devinit.org/what-we-do/what-we-are-working-on/strengthening-knowledge-humanitarian-assistance/

We were getting 30-140 redirects a month, so it would be good have this fixed by the end of April. 

Can you tag EmmaW when she's on the repository so she can check this and sign it off?

Thanks!

**Comment by SimonMurphyDI** [comment]
@edwinmp hold on a moment on this, might be a suggestion from Anna on how we do this

**Comment by akmiller01** [comment]
We had a quick video call, where he said eventually he wants it to be a redirect to a landing page with some noindex JS embedded in it. So I was testing that here: https://dev.devinit.org/landing/

But that's on hold until he's back. And until then the redirect should just be 302.

**Comment by SimonMurphyDI** [comment]
@akmiller01 this looks great. it would be ideal if you could remove the 'Share' feature (as we don't want people sharing it) but apart from that this is OKed. 

Cheers!

**Comment by akmiller01** [comment]
If you're seeing a share button, we might need to cross-browser test the adhoc JS I inserted into the page, as for me on Chrome it is removing the Share feature:

![image](https://user-images.githubusercontent.com/2836840/167451258-23a1a103-c90a-4683-8301-cc1780dfff88.png)

**Comment by SimonMurphyDI** [comment]
No my bad - I need to leave that interactive chart in (even though you can't see it in the final page)
So all good

**Comment by akmiller01** [comment]
Ah yeah, here's the JS the interactive injects into the page:

```
# Insert noindex directive into page header so Search Engines don't index the page
var metaRobots = document.createElement('meta');
metaRobots.setAttribute('name', 'robots');
document.head.appendChild(metaRobots);
metaRobots.setAttribute('content', 'noindex');

# Remove white box caused by this code
var thisChart = document.querySelector('div[data-title="Noindex"]');
thisChart.remove();

# Remove share button
var shareBox = document.querySelector('.share').parentElement;
shareBox.remove();

# Remove chapter nav
var chapterNav = document.querySelector('.js-chapter-nav-simple');
chapterNav.remove();
```

**Comment by akmiller01** [comment]
What would you like the URL of the production page to be @SimonMurphyDI ? I know you wanted something unassuming that people wouldn't stumble upon accidentally. Is `/landing` unassuming enough?

**Comment by SimonMurphyDI** [comment]
Hi Alex, could we have:

/welcome

**Comment by SimonMurphyDI** [comment]
Thanks, Alex - I've got the page ready as a draft. Anything you now need to do, or can I just push it live?

**Comment by SimonMurphyDI** [comment]
Thanks Alex, we've pushed live. Can you do your bit tomorrow earlier in your day so we can check it works?

**Comment by edwinmp** [comment]
@SimonMurphyDI @akmiller01 any update on this?

**Comment by akmiller01** [comment]
Yes, I had to move the DNS records for globalhumanitarianassistance.org to the Cloudflare free tier on the di-devs account to set up some page rules. But now the home page and all other pages redirect to /welcome, all old reports redirect to their proper places, like http://www.globalhumanitarianassistance.org/report/gha-report-2015 and all old documents redirect as well like http://www.globalhumanitarianassistance.org/wp-content/uploads/2012/04/Private-funding-an-emerging-trend.pdf

So I think this is done, but I'll leave it to @SimonMurphyDI to close.
